alvr/client_openxr/src/stream.rs

```diff
@@ -102,11 +102,7 @@ impl StreamContext {
             {
                 alvr_client_core::try_get_permission("com.oculus.permission.EYE_TRACKING")
             }
-            if config.combined_eye_gaze
-                && matches!(
-                    platform,
-                    Platform::Pico4 | Platform::PicoNeo3 | Platform::PicoNeo3Link
-                )
+            if config.combined_eye_gaze && matches!(platform, Platform::Pico4 | Platform::PicoNeo3)
```

Comment:
This seems like you're missing the commit adding Pico 3 Link support

Reply:
No, I manually reverted most of the changes because it was not the right approach. PN3L is not really a separate platform from PN3. You can see the first commit in this PR

Reply:
yep, I always rebase merge when pushing to zarik-staging